packages/terra-card/docs/README.md

```diff
@@ -29,7 +29,13 @@ Or we can selectively add padding to child elements
 	</Card.Body>
 </Card>
 
-
+// Card Theming
```

Comment:
I'd remove this card theming example. Theming will be covered in the theme provider documentation and then we'll add theming docs to the [getting started page](https://terra-ui.herokuapp.com/getting-started) as well.